src/main/java/ListOfWishlistsPage.java

```diff
@@ -0,0 +1,168 @@
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.awt.event.ComponentAdapter;
+import java.awt.event.ComponentEvent;
+
+public class ListOfWishlistsPage extends JFrame {
+    private JPanel leftPanel;
+    private JPanel upperPanel;
+    private JPanel headerPanel;
+    private JButton signoutButton;
+    private JLabel welcomeLabel;
+    private JLabel usernameLabel;
+    private JPanel upperFooterPanel;
+    private JPanel lowerPanel;
+    private JLabel listOfWishlistsLabel;
+    private JPanel wishlistPanel;
+    private JPanel outerWLPanel;
+    private JPanel lowerFooterPanel;
+    private JButton addWishlistButton;
+
+    private JButton dummyButton;
+
+    public JPanel getMainPanel() {
+        return leftPanel;
+    }
+
+    public JLabel createLabel(String text, Font font, Color color) {
+        JLabel label = new JLabel(text);
+        label.setFont(font);
+        label.setForeground(color);
+        return label;
+    }
+
+    public JButton createButton(String text, int prefWidth, int prefHeight, Color background, Color foreground, Font font) {
+        JButton button = new JButton();
+        button.setText(text);
+        button.setFont(font);
+        button.setBorder(new RoundedBorder(20));
+        if (prefHeight > 0 && prefWidth > 0) {
+            button.setPreferredSize(new Dimension(prefWidth, prefHeight));
+        }
+        button.setBackground(background);
+        button.setForeground(foreground);
+        return button;
+    }
+
+    public ListOfWishlistsPage() {
+        super("My Wishlists");
+        setLayout(null);
+        setSize(400, 638);
+        //setResizable(false);
+
+        this.addComponentListener(new ComponentAdapter() {
+            public void componentResized(ComponentEvent e) {
+                JFrame f = (JFrame)e.getSource();
+//                if (constHeight==0) {
+//                    constHeight = f.getHeight();
+//                }
+                f.setSize(400, f.getHeight());
```

Comment:
Is this code needed?

Reply:
Thanks for taking the time to catching all these bits of code I need to clean up! Truth be told, I still need some of these bits for reference in case, but your right about everything else. I'll be sure to fix what you suggested tonight so we can keep working on wiring everything up tomorrow. 

Cheers